packages/vscode-extension/src/webview/hooks/useFatalErrorAlert.tsx

```diff
@@ -90,12 +88,13 @@ const noErrorAlert = {
 
 function createBuildErrorAlert(
   buildErrorDescriptor: BuildErrorDescriptor,
-  deviceSessionsManager: DeviceSessionsManagerInterface,
   hasSelectedScheme: boolean,
   xcodeSchemes: string[]
 ) {
+  const { project } = useProject();
```

Comment:
You cannot use hooks on non-hooks, pass either `project` or `onReload` to this function.